main.go

```diff
@@ -254,6 +298,16 @@ func main() {
 
 	config.Init()
 
+	currentVersion := config.Version
+	latestVersion, err := getLatestReleaseVersion(repoOwner, repoName)
+
+	if err != nil {
+		ui.Errorf("Failed to fetch updates: %s", err)
+		os.Exit(1)
+	}
```

Comment:
We probably don't want to fail here. Best to be optimistic, log the alert "Failed to check latest CLI version" as a warning, and continue.

Reply:
Addressed!